test/functional/test_framework/test_node.py

```diff
@@ -231,7 +234,14 @@ def is_node_stopped(self):
         return True
 
     def wait_until_stopped(self, timeout=BITCOIND_PROC_WAIT_TIMEOUT):
-        wait_until(self.is_node_stopped, timeout=timeout)
+        i = 0
+        while i < 2:
+            try:
+                i = i + 1
+                wait_until(self.is_node_stopped, timeout=timeout)
+            except TimeoutError as e:
+                pass
+
```

Comment:
What is the intention behind this addition of logic?

Reply:
this was to double the timeout period for stopping a node. Some tests were failing because the node did not stop within the given time.